main.py

```diff
@@ -1,10 +1,72 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
+def create_or_update_race(race_data: dict) -> Race:
+    race_description = race_data.get("description")
+    if race_description is None:
+        return (Race.objects.get_or_create(name=race_data["name"]))[0]
+    return (Race.objects.get_or_create(
+        name=race_data["name"],
+        description=race_description)
+    )[0]
+
+
+def create_or_update_guild(guild_data: dict) -> Guild | None:
```

Comment:
`create_or_update_guild`? What exactly are you updating here?

Reply:
description